tests/modules/test_structureset.py

```
@@ -0,0 +1,161 @@
+import pytest
+import numpy as np
+from unittest.mock import MagicMock, patch
+from typing import Dict, List
```

Ruff (0.8.2)

`typing.Dict` imported but unused

Remove unused import

(F401)